certified-connectors/Verified/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2681 @@
+          "description": "Attachment Num. Max"
+        },
+        "numMin": {
+          "type": "integer",
+          "title": "Num. Min",
+          "description": "Attachment Num.Min"
+        },
+        "hash": {
+          "type": "integer",
+          "title": "Hash",
+          "description": "Attachment Hash"
+        }
+      }
+    },
+    "Template": {
+      "type": "object",
+      "title": "Template",
+      "properties": {
+        "type": {
+          "type": "string",
+          "title": "Type",
+          "description": "Template Type"
+        },
+        "html": {
+          "type": "string",
+          "title": "HTML"
+        },
+        "pdfHtml": {
+          "type": "string",
+          "title": "PDFHTML"
+        },
+        "id": {
+          "type": "string",
+          "title": "Id"
+        },
+        "uid": {
+          "type": "string",
+          "title": "UID"
+        },
+        "namespace": {
+          "type": "string",
+          "title": "Namespace"
+        },
+        "version": {
+          "type": "integer",
+          "title": "Version"
+        },
+        "created": {
+          "type": "string",
+          "title": "Created"
+        },
+        "modified": {
+          "type": "string",
+          "title": "Modified"
+        },
+        "published": {
+          "type": "boolean",
+          "title": "Modified"
+        },
+        "completed": {
+          "type": "boolean",
+          "title": "Completed"
+        },
+        "aborted": {
+          "type": "boolean",
+          "title": "Aborted"
+        }
+      }
+    },
+    "DefaultValues": {
+      "type": "object",
+      "properties": {
+        "signer": {
+          "$ref": "#/definitions/Person"
+        },
+        "reviewer": {
+          "$ref": "#/definitions/Person"
+        }
+      },
+      "description": "defaultValues"
+    },
+    "Error": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "integer",
+          "title": "Code",
+          "description": "Error Code"
+        },
+        "message": {
+          "type": "string",
+          "title": "Message",
+          "description": "Error Message"
+        }
+      }
+    }
+  },
+  "parameters": {
+    "Token": {
+      "name": "token",
+      "in": "header",
+      "type": "string",
+      "required": true,
+      "description": "Token received from the 'Generate security token' action",
+      "x-ms-visibility": "important",
+      "x-ms-summary": "Token"
+    },
+    "ContentType": {
+      "name": "Content-Type",
+      "in": "header",
+      "type": "string",
+      "required": true,
+      "description": "Content-Type",
+      "x-ms-visibility": "internal",
+      "x-ms-summary": "Content-Type of the request",
+      "default": "application/json"
+    },
+    "XNamespace": {
+      "name": "x-namespace",
+      "in": "header",
+      "type": "string",
+      "required": true,
+      "description": "",
```

Comment:
@BenediktBergmann Empty property values can be removed until unless you have a specific reason to keep them.